public/test.json5

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
This file should be removed :)

In order, to compile only one sample, you can comments line in the `samples.json5` with `//` if I remember well.

Or we can add `test.json5` to gitignore :)

Reply:
Ah yes I added test.json5 to .ignore in several of my branches 😉 But obviously not in this one.
Change done :)